app/mutations/useErrorMutation.ts

```diff
@@ -0,0 +1,52 @@
+// import { useContext } from "react";
+
+// const useMutation = (errorMessage) => {
+//   //on error
+//   //alert sentry
+//   //set the context to the errorMessage
+// };
+
+// export default useMutation;
+
+import { ErrorContext } from "contexts/ErrorContext";
+// import { CacheConfigWithDebounce } from "lib/relay/debounceMutationMiddleware";
+import { useContext } from "react";
+import { useMutation } from "react-relay";
+import { commitMutation as baseCommitMutation } from "relay-runtime";
+import {
+  Disposable,
+  GraphQLTaggedNode,
+  IEnvironment,
+  MutationConfig,
+  MutationParameters,
+} from "relay-runtime";
+import * as Sentry from "@sentry/nextjs";
+
+export default function useErrorMutation<TMutation extends MutationParameters>(
```

Comment:
A bit more verbose, but I think it's clearer.
```suggestion
export default function useMutationWithErrorMessage<TMutation extends MutationParameters>(
```